.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run update-readme && npm run format && npx lint-staged && npm run check-deps
```

Comment:
Wont this be a bit much? Or is it quite fast to run those "all files" hooks?

Reply:
Well, all of them where here before, I just move them to new config file for husky.
On my machine it took several seconds to complete. It is noticeable, but not annoying, IMO.